src/appMain/main.cc

```diff
@@ -135,7 +135,8 @@ int32_t main(int32_t argc, char** argv) {
   // Logger initialization
   INIT_LOGGER("log4cxx.properties", profile_instance.logs_enabled());
 
-  threads::Thread::SetNameForId(threads::Thread::CurrentId(), "MainThread");
+  threads::Thread::SetNameForId(threads::Thread::CurrentId(),
```

Comment:
not that this is a bad thing, but couldn't we just use the proper process name in the script instead of doing this change? pgrep/pkill smartDeviceLinkCore would work fine.